pkg/api/v1/register.go

```diff
@@ -23,8 +23,11 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
+// GroupName is the group name use in this package
+const GroupName = api.GroupName
```

Comment:
Yes, we are a ways away from the goal state. That conversion stuff needs to
be moved to its own package.
On Dec 8, 2015 11:49 AM, "David Eads" <notifications@github.com> wrote:

> In pkg/api/v1/register.go
> <https://github.com/kubernetes/kubernetes/pull/18353#discussion_r47003712>
> :
>
> > @@ -23,8 +23,11 @@ import (
> >  	"k8s.io/kubernetes/pkg/runtime"
> >  )
> >
> > +// GroupName is the group name use in this package
> > +const GroupName = api.GroupName
>
> Yes-- if we had an import tree test, this would have broken it :)
> pkg/apis/x/v1 may not import pkg/apis/x.
>
> In fairness, this breaks it right here:
> https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/conversion.go#L23
> . We seem to be rather far from where you want to be.
>
> Will update.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/18353/files#r47003712>.
>